darwin/torch/utils.py

```diff
@@ -36,13 +42,37 @@ def convert_segmentation_to_mask(segmentations: List[Segment], height: int, widt
 
 def polygon_area(x: np.ndarray, y: np.ndarray) -> float:
     """
-    Returns the area of the input polygon, represented with two numpy arrays
-    for x and y coordinates.
+    Returns the area of the input polygon, represented by two numpy arrays for x and y coordinates.
+
+    Parameters
+    ----------
+    x : np.ndarray
+        Numpy array for x coordinates.
+    y : np.ndarray
+        Numpy array for y coordinates.
+
+    Returns
+    -------
+    float
+        The area of the polygon.
     """
     return 0.5 * np.abs(np.dot(x, np.roll(y, 1)) - np.dot(y, np.roll(x, 1)))
 
 
-def collate_fn(batch) -> Tuple:
+def collate_fn(batch: Iterable[Any]) -> Tuple:
```

Comment:
You can switch `Any` for `Tuple` here. The idea of the `collate_fn` is to turn a list of tuples into a tuple of lists.

Reply:
fixed !